README.rst

```diff
@@ -98,227 +108,31 @@ You are recommended to update pip before installing aqtinstall.
-
-
-Example: Install examples, doc and source:
-
-.. code-block:: console
-
-    py -m aqt examples 5.15.0 windows desktop -m qtcharts qtnetworkauth
-    py -m aqt doc 5.15.0 windows desktop -m qtcharts qtnetworkauth
-    py -m aqt src 5.15.0 windows desktop
-
-
-Example: Install Web Assembly for Qt5
-
-.. code-block:: console
-
-    aqt install 5.15.0 linux desktop wasm_32
-
-
-Example: Install an Install FrameWork (IFW):
-
-.. code-block:: console
-
-    aqt tool linux desktop tools_ifw
-
-
-Example: Install vcredist:
-
-.. code-block:: console
-
-    py -m aqt tool windows desktop tools_vcredist
-    .\Qt\Tools\vcredist\vcredist_msvc2019_x64.exe /norestart /q
-
-
-Example: Install MinGW on Windows
-
-.. code-block:: console
-
-    py -m aqt tool -O c:\Qt windows desktop tools_mingw qt.tools.win64_mingw810
-    set PATH=C:\Qt\Tools\mingw810_64\bin
-
-
-Example: Install Qt6 for android
-
-.. code-block:: console
-
-    aqt install -O qt 6.1.0 linux desktop
-    aqt install -O qt 6.1.0 linux android android_armv7
-    qt/6.1.0/android_armv7/bin/qmake -query
-
-
-Example: Install Qt6 for ios/mac
-
-.. code-block:: console
-
-    aqt install -O qt 6.1.0 mac desktop
-    aqt install -O qt 6.1.0 mac ios ios
-    qt/6.1.0/ios/bin/qmake -query
-
-
-Example: Show help message
-
-.. code-block:: console
-
-    aqt help
-
-
-Environment Variables
----------------------
-
-It is users task to set some environment variables to fit your platform such as
-
-
-.. code-block:: bash
-
-   export PATH=/path/to/qt/x.x.x/clang_64/bin/:$PATH
-   export QT_PLUGIN_PATH=/path/to/qt/x.x.x/clang_64/plugins/
-   export QML_IMPORT_PATH=/path/to/qt/x.x.x/clang_64/qml/
-   export QML2_IMPORT_PATH=/path/to/qt/x.x.x/clang_64/qml/
-
-aqtinstall never do it for you because not to break multiple installation versions.
-
-
-
-Supported CI platforms
-----------------------
-
-There are no limitation for CI platform but currently it is tested on Azure Pipelines and Github actions.
-If you want to use it with Github actions, `install_qt`_ action will help you.
-If you want to use it with Azure Pipelines, blog article `Using Azure DevOps Pipelines with Qt`_ may be informative.
-
-
-(Advanced) Force dependency
----------------------------
-
-(Here is a note for advanced user who knows python/pip well.)
-
-When you have a trouble on your (minor) platform to install aqtinstall's dependency,
-you can force dependencies and its versions (not recommended for ordinary use).
-You can run `pip` to install individual dependencies in manual and install aqtinstall with `--no-deps`.
-
-Example:
-^^^^^^^^
-
-Avoid installation of py7zr, python 7zip library, and force using external 7z command to extract archives.
-
-.. code-block:: console
-
-    pip install -U pip
-    pip install requests==2.25.1 semantic_version texttable
-    pip install --no-deps aqtinstall
-    python -m aqt --external /usr/local/bin/7z install 5.15.2 linux desktop
+    aqt install-qt windows desktop 6.2.0 win64_mingw81
```

Comment:
```suggestion
    aqt install-qt windows desktop 6.2.0 win64_mingw81 -m all
 
The optional `-m all` argument installs all the modules available for Qt 6.2.0; you can leave it off if you don't want the modules.

To install Qt 6.2.0 with the modules 'qtcharts' and 'qtnetworking', you can use this command (note that the module names are lowercase):

.. code-block:: console

    aqt install-qt windows desktop 6.2.0 win64_mingw81 -m qtcharts qtnetworking

```

This is a continuation of the previous suggestion; please ignore it if you don't think we need to discuss modules in the README.